src/main/webapp/recipe.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Search by recipe</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body> 
+    <input type="number" id="num-recipe" name="num-recipe" value="5738"></input>
+    <br></br>
+    <button onclick="getRecipe()">Search for recipe</button>
+    <h1 id="recipe-info"></h1>
+    <input type="text" id="dish-name" name="dish-name"></input>
+    <br></br>
```

Comment:
Is this just for testing or is this a recipe-search functionality you are adding?

Reply:
Just for testing, the two current servlets are being combined into one for easier use